environment/src/main/java/com/sequenceiq/environment/audit/flow/CreationEnvironmentFlowOperationAuditEventNameConverter.java

```diff
@@ -0,0 +1,33 @@
+package com.sequenceiq.environment.audit.flow;
+
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.audit.model.AuditEventName;
+import com.sequenceiq.cloudbreak.structuredevent.event.cdp.CDPStructuredFlowEvent;
+
+@Component
+public class CreationEnvironmentFlowOperationAuditEventNameConverter implements EnvironmentFlowOperationAuditEventNameConverter {
```

Comment:
EnvironmentCreationFlowOperation.... would sound better to me, what do yo think?